recipes/thrift/all/patches/cmake.diff

```diff
@@ -5,15 +5,15 @@
  
  # If OpenSSL is not found or disabled just ignore the OpenSSL stuff
 -if(OPENSSL_FOUND AND WITH_OPENSSL)
-+if(OpenSSL_FOUND AND WITH_OPENSSL)
++if(WITH_OPENSSL)
```

Comment:
I am surprised `OpenSSL_FOUND` is not set =?

Reply:
`OpenSSL_FOUND` is set, but not `OPENSSL_FOUND`.